### DISCUSSION TITLE
测距组件的label文字怎么修改

### DISCUSSION THREAD

**Discussion by jiege15121313** [zouyaoji/vue-cesium#324] [General]
![EB234BED-E446-4924-97AB-FC38B5F69650](https://user-images.githubusercontent.com/115357712/204505991-d7dbecfb-ca7b-490c-a1e0-0d304ff2b0e9.png)
比如说我希望改成（总长度…米）应该怎么修改呢

**Comment by zouyaoji** [comment]
参考：
https://github.com/zouyaoji/vue-cesium/discussions/208
